component/file_cache/file_cache_linux_test.go

```diff
@@ -97,10 +97,10 @@ func (suite *fileCacheLinuxTestSuite) cleanupTest() {
 	os.RemoveAll(suite.fake_storage_path)
 }
 
-func (suite *fileCacheLinuxTestSuite) TestChownNotInCache() {
+func (suite *fileCacheTestSuite) TestChmodNotInCache() {
```

Comment:
This should refer to the fileCacheLinuxTestSuite instead of the fileCacheTestSuite.

Reply:
Good catch!